demo-project/conf/base/catalog_01_raw.yml

```diff
@@ -1,14 +1,41 @@
 companies:
   type: pandas.CSVDataSet
   filepath: ${base_location}/01_raw/companies.csv
-  layer: raw
+  metadata:
+    kedro-viz:
+      layer: raw
+      preview: 5
 
 reviews:
   type: pandas.CSVDataSet
   filepath: ${base_location}/01_raw/reviews.csv
-  layer: raw
+  metadata:
+    kedro-viz:
+      layer: raw
+      preview: 10
 
 shuttles:
   type: pandas.ExcelDataSet
   filepath: ${base_location}/01_raw/shuttles.xlsx
-  layer: raw
+  metadata:
+    kedro-viz:
+      layer: raw
+      preview: 15 
+
+
+
+
+# companies:
```

Comment:
are we not going to remove these completely? if not then it might be helpful to have a comment to say why we just commented them out here